From: Josef Bacik <jbacik@redhat.com>
Date: Tue, 31 Mar 2009 09:57:53 -0400
Subject: [misc] make sure fiemap.h is installed in headers pkg
Message-id: 1238507873-25117-1-git-send-email-jbacik@redhat.com
O-Subject: [PATCH] [RHEL 5.4] make sure fiemap.h is installed on make headers_install
Bugzilla: 296951
RH-Acked-by: Steven Whitehouse <swhiteho@redhat.com>
RH-Acked-by: Eric Sandeen <sandeen@redhat.com>

This is a backport of an upstream patch, in reference to bz 296951

Include fiemap.h in header-y; it defines the interface
for the FS_IOC_FIEMAP file mapping ioctl.

Signed-off-by: Eric Sandeen <sandeen@redhat.com>
Signed-off-by: Josef Bacik <jbacik@redhat.com>

diff --git a/include/linux/Kbuild b/include/linux/Kbuild
index f75f6b3..38f3591 100644
--- a/include/linux/Kbuild
+++ b/include/linux/Kbuild
@@ -64,6 +64,7 @@ header-y += fd.h
 header-y += fdreg.h
 header-y += ftape-header-segment.h
 header-y += ftape-vendors.h
+header-y += fiemap.h
 header-y += firewire-cdev.h
 header-y += firewire-constants.h
 header-y += fuse.h
